docs/api/rest_api/rest_api_usage/rest_api_usage.md

```diff
@@ -11,6 +11,19 @@ Every interaction with the repository than you can do from back office or by usi
 
 The REST API uses HTTP methods (such as `GET` and `PUBLISH`), and HTTP headers to specify the type of request.
 
+## OpenAPI support
+
+The REST API is built on top of [API Platform](https://api-platform.com/docs/symfony/) and meets the [OpenAPI](https://www.openapis.org/) standard.
+
+You can downlaod the OpenAPI specification from the [REST API Reference](/api/rest_api/rest_api_reference/rest_api_reference.html), or generate it for your project by running one of the command below:
+
+``` bash
+php bin/console ibexa:openapi --output=spec.json # JSON output
+php bin/console ibexa:openapi --yaml --output=spec.yaml # YAML output
```

Comment:
Re-reading this, I would use the [recommended name](https://spec.openapis.org/oas/v3.1.0.html#document-structure):

```suggestion
php bin/console ibexa:openapi --output=openapi.json # JSON output
php bin/console ibexa:openapi --yaml --output=openapi.yaml # YAML output
```

Side note, I have always redirect the output to a file instead of using the `--output` option or its `-o` shortcut. The option probably ease command embedding/wrapping (like when you need to path it to a container).